packages/fhir-keycloak-user-management/src/components/UserList/Viewdetails/index.tsx

```diff
@@ -100,14 +101,27 @@ export const UserDetails = (props: UserDetailProps) => {
   }
 
   const { id, firstName, lastName, username, email, emailVerified, enabled, attributes } = user;
-  const userDetails = {
-    [t('Id')]: id,
-    [t('First Name')]: firstName,
-    [t('Last Name')]: lastName,
-    [t('Username')]: username,
-    [t('Email')]: email,
-    [t('Verified')]: emailVerified ? t('True') : t('False'),
-  };
+
+  const userDetails =
+    attributes?.phoneNumber && attributes?.nationalId
+      ? {
+          [t('Id')]: id,
+          [t('First Name')]: firstName,
+          [t('Last Name')]: lastName,
+          [t('National ID')]: attributes?.nationalId,
+          [t('Phone Number')]: attributes?.phoneNumber,
```

Comment:
nit: could we re-use the same spread pattern for optional computed properties like shown below:
`...(nationalId ? { nationalId: [nationalId] } : {}),`